spec.bs

```diff
@@ -18,3 +18,122 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 Assume Explicit For: yes
 </pre>
+
+
+<h2 id="advertisements">Advertisements</h2>
+
+An {{AuctionAd}} contains the following:
+
+* A <dfn for=AuctionAd dfn-type="dict-member">renderUrl</dfn> .
+* An optional <dfn for=AuctionAd dfn-type="dict-member">metadata</dfn> which is text containing opaque JSON encoded data.
+
+<xmp class="idl">
+dictionary AuctionAd {
+  required USVString renderUrl;
+  any metadata;
+};
+</xmp>
+
+
+<h2 id="advertisement-interest-groups">Advertisement Interest Groups</h2>
+
+An <dfn export>advertisement interest group</dfn>
+is a [=dictionary=] with the following:
+ 
+* An <dfn export for="AuctionAdInterestGroup">expiration</dfn> point in time.
+
+* An <dfn export for="AuctionAdInterestGroup">owner</dfn> [=origin=].

```

Comment:
This line says "A blah type" but the next line says "A blah, a type", should we make all the lines read the same?

Reply:
Yes.

Reply:
The "blah type" reads better in some cases, but consistency is probably more important.